### PR TITLE
Build fails out of the box due to a missing webpack-cli dev dependenc…

### PR DESCRIPTION
**Pull request by TCourtneyOwen**
…y and other missing webpack configs #389

- Updated webpack to v.4.21.1
- Updated other packages as neccessary to work with latest webpack
- Verified React template now builds, that "start" command now successfully starts webpack-dev-server and that the addin now works

### CODE REVIEW

File: src/app/templates/ts/react/package.json [hunk 1]
```diff
@@ -27,11 +27,11 @@
         "@types/webpack": "2.2.12",
         "@types/webpack-dev-server": "1.12.6",
         "autoprefixer": "6.7.7",
-        "copy-webpack-plugin": "4.0.1",
+        "copy-webpack-plugin": "4.6.0",
         "css-loader": "0.27.3",
-        "extract-text-webpack-plugin": "2.1.0",
+        "extract-text-webpack-plugin": "4.0.0-beta.0",
```
[the remark this answers is not in the document]
Comment (TCourtneyOwen) [reply src/app/templates/ts/react/package.json]: Yes, the beta version  is required to address an error that surfaced after updating the webpack version. I did some research and the recommended fix for the particular error is to use the 4.0.0-beta.0 version of the extract-text-webpack-plugin package

File: src/app/templates/ts/react/package.json [hunk 2]
```diff
@@ -41,7 +41,8 @@
         "tslint": "^5.9.1",
         "ts-loader": "^3.5.0",
         "typescript": "^2.7.2",
-        "webpack": "^3.11.0",
+        "webpack": "^4.21.1",
```
Comment (TCourtneyOwen) [inline src/app/templates/ts/react/package.json]: I updated the webpack version locally to 4.27.1

**Comment by SeanKilleen** [comment]
Awesome -- thanks for the quick turnaround! Does this go live automatically, or is there a specific release I'll need to hold off until?

**Comment by TCourtneyOwen** [comment]
I published a new version of generator-office with the fix. You just need to update to the latest version and then recreate your React project.

**Pull request merged**